RecoParticleFlow/PFClusterProducer/python/particleFlowClusterOOTECAL_cff.py

```diff
@@ -1,5 +1,7 @@
 import FWCore.ParameterSet.Config as cms
-from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cff import *
 
 particleFlowClusterOOTECAL = particleFlowClusterECAL.clone()
+particleFlowClusterOOTECAL.energyCorrector.applyMVACorrections = cms.bool(True)
+particleFlowClusterOOTECAL.energyCorrector.maxPtForMVAEvaluation = cms.double(90.)
```

Comment:
why is this edit needed?

this edit will override the run2_ECAL_2017 modifier settings
and IIUC, for this setup the OOT needs the new corrector to be consistent with everything else used in 2017

In any case, please drop the type specifications on the right hand side